packages/jest-runner/src/configLoaders/ReactScriptsTSJestConfigLoader.ts

```diff
@@ -18,10 +19,7 @@ export default class ReactScriptsTSJestConfigLoader implements JestConfigLoader
 
       // Create the React configuration for Jest
       const jestConfiguration = this.createJestConfig(reactScriptsTsLocation);
-
-      // Set test environment to jsdom (otherwise Jest won't run)
       jestConfiguration.testEnvironment = 'jsdom';
```

Comment:
@simondel this made the test succeed

Reply:
Well it turns out `react-scripts` does use `'jsdom'` as a default (https://github.com/facebook/create-react-app/pull/5074) but `react-scripts-ts` was forked *before* that change was made and it hasn't really been updated for two years...